### PR TITLE
Warn about missing doctype declaration

### PR DESCRIPTION
**Pull request by tomivirkki**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-grid/782)

### CODE REVIEW

**Comment by manolo** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---

*[vaadin-grid.html, line 223 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-grid/782#-KeczL6OCh8WFcTjetkl:-KeczL6OCh8WFcTjetkm:b6fdsuv) ([raw file](https://github.com/vaadin/vaadin-grid/blob/604f276220013dbc2242844a0f9f30800544048b/vaadin-grid.html#L223)):*
> ```HTML
> 
>       if (document.doctype === null) {
>         console.warn('<vaadin-grid> requires <!DOCTYPE html> declaration');
> ```

I would rephrase this to:

`<vaadin-grid> requires the 'Standards mode' declaration. Please add <!DOCTYPE html> to the html page.`

---

*[test/doctype.html, line 5 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-grid/782#-Ked-9-3fkWxogQUnH5T:-Ked-9-3fkWxogQUnH5U:b-uime80) ([raw file](https://github.com/vaadin/vaadin-grid/blob/604f276220013dbc2242844a0f9f30800544048b/test/doctype.html#L5)):*
> ```HTML
> <head>
>   <meta charset="UTF-8">
>   <title>iron-list test</title>
> ```

`<title>doctype test</title>`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/782)*

**Comment by tomivirkki** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[vaadin-grid.html, line 223 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-grid/782#-KeczL6OCh8WFcTjetkl:-Ked5Iv8JPMTarBcFCgc:b-896fix) ([raw file](https://github.com/vaadin/vaadin-grid/blob/604f276220013dbc2242844a0f9f30800544048b/vaadin-grid.html#L223)):*
<details><summary><i>Previously, manolo (Manuel Carrasco Moñino) wrote…</i></summary><blockquote>

I would rephrase this to:

`<vaadin-grid> requires the 'Standards mode' declaration. Please add <!DOCTYPE html> to the html page.`

</blockquote></details>

Done.

---

*[test/doctype.html, line 5 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-grid/782#-Ked-9-3fkWxogQUnH5T:-Ked24OlnS8kjZrlqEIe:b-896fix) ([raw file](https://github.com/vaadin/vaadin-grid/blob/604f276220013dbc2242844a0f9f30800544048b/test/doctype.html#L5)):*
<details><summary><i>Previously, manolo (Manuel Carrasco Moñino) wrote…</i></summary><blockquote>

`<title>doctype test</title>`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/782)*

**Comment by manolo** [comment]
:lgtm:

---

Reviewed 1 of 3 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/782#-:-KedB38kO6kiKDNkLZvO:bnfp4nl)*

**Comment by limonte** [comment]
LGTM
